.changeset/tender-buttons-refuse.md

```diff
@@ -0,0 +1,5 @@
+---
+'@datadog/ui-extensions-sdk': minor
```

Comment:
I think the reason CI is failing is because we have to release all the the `@datadog/ui-extensions-*` packages in lock step so you can either [add an additional changeset](https://github.com/DataDog/apps/runs/5852676304?check_suite_focus=true#step:7:16), or add the react package to this changeset. Either way should get that step passing.

```suggestion
'@datadog/ui-extensions-react': minor
'@datadog/ui-extensions-sdk': minor
```

Reply:
Oh interesting; so, I guess, from what you're saying I just didn't pick all the right `ui-extensions-*` packages when I ran `yarn changeset add`?